pywbem/_subscription_manager.py

```diff
@@ -1162,26 +1170,28 @@ def _create_destination(self, server_id, dest_url, owned):
         # Validate server_id
         server = self._get_server(server_id)
 
+        svr_name = self._get_systemname(server_id)
```

Comment:
Can we call this variable "system_name" instead?

Reply:
Since the medthod does not exist, DONE